proc-macros/tests/ui/correct/basic.rs

```diff
@@ -111,6 +122,8 @@ async fn main() {
 	let client = WsClientBuilder::default().build(&server_url).await.unwrap();
 
 	assert_eq!(client.async_method(10, "a".into()).await.unwrap(), 42);
+	assert_eq!(client.rename_params(256, true).await.unwrap(), 128);
+	assert_eq!(client.rename_params(256, false).await.unwrap(), 256);
```

Comment:
you can just add two additional tests here with object params.

```rust
	let mut params = ObjectParams::new();
	params.insert("type", 256).unwrap();
	params.insert("halfType", false).unwrap();

	assert_eq(client.request::<u16, ObjectParams>("rename_params", params).await.unwrap(), 128);
```

```rust
	let mut params = ObjectParams::new();
	params.insert("type", 256).unwrap();
	params.insert("halfType", true).unwrap();

	assert_eq(client.request::<u16, ObjectParams>("rename_params", params).await.unwrap(), 256);
```

Then remove the extra test file that you added below "proc-macros/tests/ui/correct/rename_fn_params.rs"

Reply:
done